pages/index.tsx

```diff
@@ -248,6 +249,17 @@ function Home() {
                 }}
               />
             </FieldSet>
+            <FieldSet>
```

Comment:
I think we have an SSN mask component?

Reply:
oooh do you have a link

Reply:
No. @mary-snow do you know if we have a react mask component?

Reply:
I did an ssn mask for prove prefill! You can check it out here and there's some helpers used that come from a helper file. https://github.com/UseAlloy/Alloy/blob/master/sdk/frontend/src/containers/Prove/Prefill/index.tsx#L93

Reply:
added!

Reply:
Wondering if you also wanted to add the `format` function to the masked ssn. So that way it's in a XXX-XX-XXXX format

Reply:
added !